themes/sap_glow/css_variables.css

```diff
@@ -1110,11 +1110,11 @@
   --udexTypographyBodyLFontWeight: 400;
   --udexTypographyBodyLLineHeight: 30px;
   --udexTypographyBodyXLFontSize: 24px;
-  --udexTypographyBodyXLFontFamily: "72", "Arial", "Helvetica", "sans-serif";
+  --udexTypographyBodyXLFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
   --udexTypographyBodyXLFontWeight: 400;
   --udexTypographyBodyXLLineHeight: 36px;
   --udexTypographyBodyXXLFontSize: 28px;
-  --udexTypographyBodyXXLFontFamily: "72", "Arial", "Helvetica", "sans-serif";
+  --udexTypographyBodyXXLFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
```

Comment:
We shouldn't touch this file, but the change (to be made by SAP) is requested here: https://github.com/urfuwo/hlx-test/issues/71. So, all fine.

Reply:
Yes I reverted the change sine @ok has provided an updated version of design token at https://github.com/urfuwo/hlx-test/issues/71#issuecomment-1910600148 .